docs/usage.md

```diff
@@ -75,10 +76,10 @@ You can now start customising the configuration you are going to run TopoStats w
 ones you may want to change are....
 
 * `base_dir` (default: `./`) the directory in which to search for scans. By default this is `./` which represents the
-  directory from which `run_topostats` is called and it is good practice to have one configuration file per batch of
+  folder from which `run_topostats` is called and it is good practice to have one configuration file per batch of
```

Comment:
Directory is a more generic term that aligns with instructions on `c`hanging `d`irectory (i.e. `cd` command).

Reply:
Directory however is an uncommon term to users unfamiliar with command line. Hence why I thought I'd introduce it here and only here.